newGame.html

```diff
@@ -0,0 +1,41 @@
+<script>
+  (function() {
+    var value = ['two','three','four','five','six','seven','eight','nine','ten','jack','queen','king','ace'];
+    var rand = value[Math.floor(Math.random() * value.length)];
+
+
```

Comment:
Remember to remove the white space/line breaks in your code and consolidate as much as possible, and within reason. Think of paragraphs of text, and using line breaks to create separation in your code. You should contain relative "ideas" (blocks of code) in a paragraph.

Also, below this comment, you want to remove comments from your commits in professional practice. It's good to get into that habit early on!